Packs/Okta/Scripts/IAMInitOktaUser/README.md

```diff
@@ -0,0 +1,4 @@
+Generates a password and sets an Okta user account with that password.
```

Comment:
please generate a doc using the sdk command (or copy the AD one and change manually)